docs/src/pages/components/data-grid/demo/demo.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Data Grid - Demo
+components: DataGrid, XGrid
+---
+
+# Data Grid - Demo
+
+<p class="description">Use the demos below to explore the available features</p>
+
+## Full featured
```

Comment:
useless block I think

Reply:
I added it because initially there were 2 demos but for now, maybe we can remove it and add those headers once we have 1+ demos on the page (also the right nav is gone).